include/msgbuffer.h

```diff
@@ -5,22 +5,13 @@
  * @author Amy Wagoner <arwagoner@gmail.com> (adaptation to Gazebo)
  */
 /*
-/*
- * Copyright (C) 2012-2017 Open Source Robotics Foundation
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *     http://www.apache.org/licenses/LICENSE-2.0
+ * libmavconn
+ * Copyright 2014,2015,2016 Vladimir Ermakov, All rights reserved.
  *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- *
-*/
+ * This file is part of the mavros package and subject to the license terms
```

Comment:
@TSC21 This doesn't work. We need this file with MAVROS attribution (NOT flagged as being part of the package) and under Apache 2.0 - or replace it entirely.

Reply:
@LorenzMeier what do you mean with "MAVROS attribution"?